src/style.css

```diff
@@ -0,0 +1,137 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --background-color-blue: #00acdc;
+  --background-color-white: #fff;
+  --dark-color-text: #060b35;
+  font-family: "Roboto", sans-serif;
+}
+
+h6,
+p {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  background-color: var(--background-color-white);
+  border-radius: 5px;
+}
+
+.card__mac-image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
+}
+
+.card__title {
+  padding: 40px 16px 0;
```

Comment:
To add external indent between elements it is better to use margin. Please fix for all cases
![image](https://github.com/mate-academy/layout_product-cards/assets/54936044/95376f7e-6c11-44b7-89e4-5b70ca1386e0)